custom_cms.sql

```diff
@@ -24,6 +24,17 @@ SET time_zone = "+00:00";
 
 -- --------------------------------------------------------
 
+--
+-- Table structure for table `customBlock`
+--
+
+CREATE TABLE `customBlock` (
```

Comment:
How are we going to identify any unique block?

Reply:
9be20c63: I have added a unique id field in table through which we can identify any unique block.